Transfer/Sources/TransferAllFundsCell.swift

```diff
@@ -88,6 +88,10 @@ final class TransferAllFundsCell: UITableViewCell {
     }
 
     func configure(setOn: Bool, _ handler: @escaping TransferAllFundsSwitchHandler) {
+        titleLabel.adjustsFontForContentSizeCategory = true
+        titleLabel.numberOfLines = 0
+        titleLabel.font = Theme.Label.captionOne
```

Comment:
This is being set from ThemeManager so remove it from here.

Reply:
Also update ``` @objc dynamic var titleLabelFont: UIFont! {
        get { return titleLabel.font }
        set {
            titleLabel.font = newValue
            amountTextField.font = newValue
        }
    }

    @objc dynamic var titleLabelColor: UIColor! {
        get { return titleLabel.textColor }
        set {
            titleLabel.textColor = newValue
            amountTextField.textColor = newValue
        }
    } ```

Reply:
Removed